### DISCUSSION TITLE
Rallypoint explanation

### DISCUSSION THREAD

**Discussion by TheLocalPub** [Vdauphin/HeartsAndMinds#1609] [Question and answer] [answered]
Hey.

So I see there's a feature to be able to place a rally point?
Which is a backpack in the arsenal?

I'm unable to find this or locate the feature.
Can you better describe this feature and it's content as currently it's quite vague.

**Comment by Vdauphin** [comment]
Hello,

> Which is a backpack in the arsenal?

Yes, it is a vanilla backpack which allow you to do that, it is not related to H&M
Did you look here: http://vdauphin.github.io/HeartsAndMinds/InGame-documentation#respawn-position

Cheers

**Reply by TheLocalPub** [reply]
I read through this but I think maybe English isn't your strongest language?
Because it still didn't make sense.

> Rallypoint: You can also use the Zeus rallypoint backpack available in Arsenal but they have some limitation. After some times or after player disconnected, rallypoint will self-destruct.

What is the "zeus rally point backpack".
You just said above it's a vanilla backpack, but which backpack/classname?

**Reply by Vdauphin** [reply, marked as answer]
> Zeus rallypoint backpack available in Arsenal

Open the Arsenal, select the backpack category, sort by mod and go to the Zeus category, click on one of them. Exit the Arsenal, use your mouse scroll wheel to select it, validate the selection
If you need help to use A3 feature I would advise you to join the A3 Discord

**Reply by TheLocalPub** [reply]
I've been part of the Arma series for as long as I can remember. I've had Arma 3 since backing it in Alpha.
I still don't know what this backpack you talk of. 

But you've given me the explanation, so tomorrow, I will do exactly this and try see what you mean, because I've *never* seen anything like this.

**Reply by TheLocalPub** [reply]
So I found the actual answer. Thanks to the A3 Discord, I asked there and the first reply gave the correct thing.
I quote:

> Any of the "Camp" backpacks, and they're from one of the expansions.
Here you go. 🙂
`"B_Respawn_Sleeping_bag_blue_F"
"B_Respawn_Sleeping_bag_brown_F"
"B_Respawn_TentDome_F"
"B_Patrol_Respawn_bag_F"
"B_Respawn_Sleeping_bag_F"
"B_Respawn_TentA_F"`
TIL there were camp backpacks. 😄 What I was remembering was something else. lol


![image](https://github.com/Vdauphin/HeartsAndMinds/assets/4511118/ce8dea93-8b7c-4641-988c-bc388204b57e)
 Those are the classnames. 
 
 In reference to "zeus", you mean't the little icon on the right next to the backpack.
 If you searched zeus you'd get no results.
